uiowa/uiowa
# [Bug] two column layouts

Since https://github.com/uiowa/uiowa/issues/3585 is still happening, I'm still curious about trying this out: https://github.com/uiowa/uiowa/issues/3585#issuecomment-941358866 to see if it makes a difference. 

# How to test

- Review code 
- Sync a site and verify that the two column layout is still an option and that there are no issues swapping the layout classes.   
- Adjust the layout definition to verify that it is using the class defined in `uiowa_core`
